core/arch/arm/include/sm/optee_smc.h

```diff
@@ -422,6 +422,44 @@
 #define OPTEE_SMC_VM_DESTROYED \
 	OPTEE_SMC_FAST_CALL_VAL(OPTEE_SMC_FUNCID_VM_DESTROYED)
 
+/*
+ * Query OP-TEE about its build configuration
```

Comment:
I'd rather simplify it by only reporting the number of supported threads, nothing more, nothing less.
If we need something more later we can add another SMC.